docs/concepts/extensions.rst

```diff
@@ -0,0 +1,256 @@
+for examples of asset yaml declarations.
+
+Override Superset Default Assets
+================================
+
+Aspects allows to override the default superset assets by creating an superset extra asset
+as explained above using the same UUID as the default asset. The UUID of the default assets
+can be found in the `assets.yaml`
+
+Changing Superset Language Settings
+===================================
+
+Superset localization is a work in progress, but you can change the default language and
+set alternate languages from the currently supported list by changing the following tutor
+configuration variables:
+
+.. code-block:: yaml
+
+    SUPERSET_DEFAULT_LOCALE: en
+    SUPERSET_SUPPORTED_LANGUAGES:
+      en:
+        flag: us
+        name: English
+      es:
+        flag: es
+        name: Spanish
+      it:
+        flag: it
+        name: Italian
+      fr:
+        flag: fr
+        name: French
+      zh:
+        flag: cn
+        name: Chinese
+      ja:
+        flag: jp
+        name: Japanese
+      de:
+        flag: de
+        name: German
+      pt:
+        flag: pt
+        name: Portuguese
+      pt_BR:
+        flag: br
+        name: Brazilian Portuguese
+      ru:
+        flag: ru
+        name: Russian
+      ko:
+        flag: kr
+        name: Korean
+      sk:
+        flag: sk
+        name: Slovak
+      sl:
+        flag: si
+        name: Slovenian
+      nl:
+        flag: nl
+        name: Dutch
+
+Where the first key is the abbreviation of the language to use, "flag" is which flag
+icon is displayed in the user interface for choosing the language, and "name" is the
+displayed name for that language. The mapping above shows all of the current languages
+supported by Superset, but please note that different languages have different levels
+of completion and support at this time.
+
+Custom Row Level Security Filters to Superset
+=============================================
+If you add new datasources, tables, or datasets to Superset, you may want to add new
+row level security filters to restrict access to that data based on things like user
+roles, or organization. To apply custom row level security filters to Superset,
+you can use the patch `superset-row-level-security`. This patch expects a list of python
+dictionaries with the following structure:
+
+.. code-block:: yaml
+
+    superset-row-level-security: |
+        {
+            "schema": "{{ASPECTS_XAPI_DATABASE}}",
+            "table_name": "{{ASPECTS_XAPI_TABLE}}",
+            "role_name": "{{SUPERSET_ROLES_MAPPING.instructor}}",
+            "group_key": "{{SUPERSET_ROW_LEVEL_SECURITY_XAPI_GROUP_KEY}}",
+            "clause": {% raw %}'{{can_view_courses(current_username(), "splitByChar(\'/\', course_id)[-1]")}}',{% endraw %}
+            "filter_type": "Regular",
+        },
+
+You can find more information about superset RLSF in the `superset documentation <https://superset.apache.org/docs/security/#row-level-security>`_.
```

Comment:
Wherever we talk about things like this (Jinja filters, roles, etc) we should add links like you did here!